packages/code-du-travail-api/src/server/routes/search/getDocumentByUrlQuery.js

```diff
@@ -21,7 +22,11 @@ const getDocumentByUrlQuery = (
     _source,
     query: {
       bool: {
-        filter: [{ term: { slug } }, { term: { source } }],
+        filter: [
+          { term: { slug } },
+          { term: { source } },
+          { term: { isPublished: true } },
```

Comment:
this should not be needed since its used by getEsReference which handle `isPublished`flag